client/state/simple-payments/product-list/actions.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import {
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE_UPDATE,
+	SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE_DELETE,
+} from 'state/action-types';
+
+export function receiveProductsList( siteId, numOfProducts, posts ) {
+	return {
+		type: SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE,
+		siteId,
+		products: posts,
+	};
+}
+
+export function receiveUpdateProduct( siteId, product ) {
```

Comment:
So for this example this would be 

```jsx
export function addProduct( siteId, product ) {
    return {
        siteId,
        type: SIMPLE_PAYMENTS_PRODUCTS_LIST_RECEIVE_UPDATE,
        product
    }
}
```